esper/controllers/commandsV2/commandsV2.py

```diff
@@ -660,7 +660,7 @@ def command(self):
         command_args_ = {
             "app_state": self.app.pargs.app_state,
             "app_version": self.app.pargs.app_version,
-            "custom_settings_config": self.app.pargs.custom_settings_config,
+            "custom_settings_config": json.loads(self.app.pargs.custom_settings_config),
```

Comment:
How big is this file? If it is too big, its probably not a good idea to load everthing to memory.
Otherwise, if this file is always a consistent/predictable size, then this is fine.

Reply:
I guess this should be fine as this is a predictable size, with an upper limit to the number of characters. 🤔 

On another note, what are the chances it could not be a **JSON**? 
Are we handling the scenario where `json.loads` fails?

Reply:
It seems like the `-cs` (or `--custom_config`) flag does not have file support. Besides, the input is translated 1:1 to command args JSON payload (subfield `custom_settings_config`), so it must be a valid JSON payload and should error out if `json.loads` fails, and I believe the resulting error should suffice, for example:
```
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 1 column 52 (char 51)
```

However, it would be a good idea to provide custom error messaging on top of this to make the error a little bit more precise, so I'll update the PR with that change.

---

Regarding the upper limit, seems like only up to 10 objects are used, based off of information from device team. So we should be good there.